uniffi_bindgen/src/bindings/gecko_js/webidl.rs

```diff
@@ -111,6 +111,8 @@ impl From<Type> for WebIDLType {
                 // https://github.com/mozilla/uniffi-rs/issues/295.
                 panic!("[TODO: From<Type>({:?})]", type_)
             }
+            Type::Timestamp => panic!("Timestamp unimplemented"),
+            Type::Duration => panic!("Duration unimplemented"),
```

Comment:
👍🏻 ; frankly I'd like to find a way for us to not have the `gecko_js` bindings in this repo right now, because they're slowly bit-rotting, but it's definitely not something to worry about in this PR.